yarn-project/noir-libs/noir-aztec/src/abi.nr

```diff
@@ -319,8 +319,8 @@ impl PublicCircuitPublicInputs {
         inputs.push_array(self.new_nullifiers);
         inputs.push_array(self.new_l2_to_l1_msgs);
 
-        // We do not include block_hash_data since it's not in the cpp hash
-        // inputs.push(self.block_hash_data.hash());
+        // We do not include block_data since it's not in the cpp hash
+        // inputs.push(self.block_data.hash());
```

Comment:
Can you add an issue for this, should be handled instead of just living as comment in here.

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/1473